toucan_data_sdk/utils/postprocess/math.py

```diff
@@ -172,6 +187,9 @@ def formula(df, *, new_column: str, formula: str):
     |   toto   |    10    |  300  | 290 |
 
     """
+    if '`' not in formula:  # OLD SYNTAX
+        formula = get_new_syntax_formula(formula)
+        logging.getLogger(__name__).warning(f"DEPRECATED: You should always use ` for your columns")
```

Comment:
Could we be a little more explicit here by showing the "before" / "after" result in the deprecation message?